Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ghcr.io/prefix-dev/pixi:0.31.0 AS build
+
+RUN apt update
+RUN apt install -y git
+
+COPY . /app
+WORKDIR /app
+
+RUN ./docker/install.sh
+
+FROM ubuntu:24.04 AS production
+
+COPY --from=build /app/.pixi/envs/default /app/.pixi/envs/default
+COPY --from=build /app/app_proxy.py /app
+COPY --from=build /app/app_config.toml /app
+COPY --from=build --chmod=0555 /entrypoint.sh /
+
+WORKDIR /app
+EXPOSE 8080
+CMD ["/entrypoint.sh"]
```

Comment:
I would specify `entrypoint.sh` as `ENTRYPOINT` and the `streamlit run` part as `CMD`. 

This allows you to execute different commands in `docker run ... custom-stuff-in-pixi-env`

See also https://tech.quantco.com/blog/pixi-production

Reply:
Looks good